atest/robot/running/for.robot

```diff
@@ -254,6 +257,80 @@ For In Range With Multiple Variables
     Should Be For Item    ${tc.kws[0].kws[2]}    \${i} = 5, \${j} = 6, \${k} = 7
     Should Be For Item    ${tc.kws[0].kws[3]}    \${i} = 8, \${j} = 9, \${k} = 10
 
+For In Zip
+    ${tc} =    Check Test Case    ${TEST NAME}
+    ${for_loop}=    Set Variable    ${tc.kws[2]}
+    Should Be For In Zip Keyword    ${for_loop}    4
+    Should Be For Item    ${for_loop.kws[0]}    \${item} = a, \${thing} = e
+    Should Be For Item    ${for_loop.kws[1]}    \${item} = b, \${thing} = f
+    Should Be For Item    ${for_loop.kws[2]}    \${item} = c, \${thing} = g
+    Should Be For Item    ${for_loop.kws[3]}    \${item} = d, \${thing} = h
+
+For In Zip With Uneven Lists
+    [Documentation]    Handling lists with different number of elements. Probably should work like in Python here and just stop when the shortest list is empty?
+    ${tc} =    Check Test Case    ${TEST NAME}
+    ${for_loop}=    Set Variable    ${tc.kws[2]}
+    Should Be For In Zip Keyword    ${for_loop}    3
+    Should Be For Item    ${for_loop.kws[0]}    \${item} = a, \${thing} = d
+    Should Be For Item    ${for_loop.kws[1]}    \${item} = b, \${thing} = e
+    Should Be For Item    ${for_loop.kws[2]}    \${item} = c, \${thing} = f
+
+For In Zip With 3 Lists
+    [Documentation]    Handling more than one list like :FOR ${a} ${b} ${c} IN ZIP ${x} ${y} ${z}.
```

Comment:
- Doc isn't very useful here. Especially when it should say "more than two lists" instead of "more than one".
- Test name should also contain "with more than two lists" rather than exact "with 3 lists".